src/utils.ts

```diff
@@ -311,6 +384,25 @@ export function trimUndefined(object) {
   return object;
 }
 
+export function getRandomUuid(): string {
```

Comment:
Do need `getRandomUuid()` in production code? Could this be moved to tests?

Reply:
We use it in one place to generate random message IDs for postMessage calls. It's not completely necessary since any random string would be okay, but it's not that long, just 17 lines.

Reply:
Ok, just want to make sure it wasn't there by mistake